client/packages/coldchain/src/Equipment/DetailView/DetailView.tsx

```diff
@@ -75,17 +75,23 @@ export const EquipmentDetailView = () => {
 
   useEffect(() => {
     if (!data) return;
-    let properties = JSON.parse(data.properties);
-    if (!properties || !ObjUtils.isObject(properties)) {
+    let assetProperties = JSON.parse(data.properties);
```

Comment:
how about instead of these lines, updating `ObjUtils`

```
import isEqual from 'lodash/isEqual';

type ParsedObject = Record<string, string | number | boolean | null>;

// Checks is input is an actual object (i.e. key-vals)
const isObject = (input: unknown): input is Record<string, unknown> =>
  typeof input === 'object' && input !== null && !Array.isArray(input);

export const ObjUtils = {
  // Performs a deep comparison between two values to determine if they are equivalent.
  isEqual,
  isObject,
  parse: (input: string | null | undefined): ParsedObject => {
    if (!input) return {};

    const maybeObject = JSON.parse(input) as ParsedObject;
    return !!maybeObject && isObject(maybeObject) ? maybeObject : {};
  },
};
```

and then 

```
    const assetProperties = ObjUtils.parse(data.properties);
    const catalogProperties = ObjUtils.parse(data.catalogProperties);
```

Reply:
[63e4ad7](https://github.com/msupply-foundation/open-msupply/pull/4001/commits/63e4ad7f0a8ebafaa70e4e43ed02141c88a3f239)